templates/vsc/js/sso-tab-naa/.gitignore

```diff
@@ -0,0 +1,18 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Please remember to update CODEOWNER file for new template folder and clean legacy templates.

Reply:
Sure. Will update in another PR.